src/components/meshes/Extrude.js

```diff
@@ -39,12 +39,12 @@ class Extrude extends MeshComponent {
   }
 
   buildGeometry(params = {}) {
-    const GConstruct = params.buffer && !params.softbody ? ExtrudeBufferGeometry : ExtrudeGeometry;
-
-    return new GConstruct(
+    const extrudeGeometry = new ExtrudeGeometry(
       params.geometry.shapes,
       params.geometry.options
     );
+
+    return params.buffer && !params.softbody ? new BufferGeometry().fromGeometry(extrudeGeometry) : extrudeGeometry;
```

Comment:
`params.softbody` should not be used. Physics is now completely separate from whs. Can you remove it, please?

-----

Good idea to use `BufferGeometry().from`. We need to do it also for other meshes. I'll make an issue.